aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -92,7 +93,7 @@ public class GlueCatalog extends BaseMetastoreCatalog
   private FileIO fileIO;
   private LockManager lockManager;
   private CloseableGroup closeableGroup;
-  private Map<String, String> catalogProperties;
+  private Map<String, String> catalogProperties = Collections.emptyMap();
```

Comment:
Nit: Unless we _need_ null values in the map, we almost always prefer shaded-guava's `ImmutableMap.of()`.